federatedscope/nlp/trainer/fedprox_trainer.py

```diff
@@ -0,0 +1,74 @@
+import logging
```

Comment:
it seems like there is no difference with the original `core/trainer/fedprox_trainer.py`. maybe we can reuse the one in core/trainer?